### DISCUSSION TITLE
Add link to documentation in helpers config flow

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#21535] [Dashboard]
set up the new history_stats helper in 2024.8 and was shown this during the config flow:

<img width="586" alt="Scherm­afbeelding 2024-08-01 om 15 23 32" src="https://github.com/user-attachments/assets/c5176343-28fe-4f4e-be2a-d4dae2ea67e7">

It would be most helpful if that Documentation was directly linked, and take the user to the online pages while keeping the flow window open.

Like this:

![image](https://github.com/user-attachments/assets/a10b04a2-b1dc-461f-a24e-ddd0abd74a44)
as was suggested in Discord #beta 

Thanks for considering

**Comment by karwosts** [comment]
Part of the way there with https://github.com/home-assistant/frontend/pull/21537
